Modules/_tkinter.c

```diff
@@ -3474,18 +3475,21 @@ PyInit__tkinter(void)
         Py_DECREF(m);
         return NULL;
     }
+    Py_DECREF(Tkapp_Type);
```

Comment:
It is the same for all other global references (otherwise why would they be global references?).

Yes, if you want to release these references, you need a module clearing function. This is not a big deal, because usually when you unload the module, you quit the program, so who cares about few dangling references?